src/components/SubNav/SubNav.tsx

```diff
@@ -0,0 +1,312 @@
+  "section.connect.primary",
+  "section.education.primary",
+  "section.locations.primary",
+  "section.research.primary",
+  "section.research-library-lpa.primary",
+  "section.research-library-schomburg.primary",
+  "section.research-library-schwarzman.primary",
+  "section.whats-on.primary",
+  "dark.brand.primary",
+  "dark.section.blogs.primary",
+  "dark.section.books-and-more.primary",
+  "dark.section.connect.primary",
+  "dark.section.education.primary",
+  "dark.section.locations.primary",
+  "dark.section.research.secondary",
+  "dark.section.research-library-lpa.primary",
+  "dark.section.research-library-schomburg.primary",
+  "dark.section.research-library-schwarzman.primary",
+  "dark.section.whats-on.primary",
+];
+export type highlightColors = typeof highlightColorsArray[number];
+
+export interface SubNavProps {
+  /**
+   * The background color to be applied to the hover and active states
+   * of the SubNavLink and SubNavButton components.
+   * This allows for customization of the action items.
+   */
+  actionBackgroundColor?: actionBackgroundColors;
+  /** Additional class name for the `SubNav` component. */
+  className?: string;
+  /**
+   * Custom color for SubNavLink, SubNavButton, and icons.
+   */
+  highlightColor?: highlightColors;
+  /**
+   * Optional unique ID for accessibility, allowing other components
+   * to reference this element.
+   */
+  id?: string;
+  /**
+   * Primary actions displayed on the left side of the SubNav.
+   * Use SubNavButton and SubNavLink components, which mirror
+   * the DS Button and Link.
+   */
+  primaryActions: React.ReactNode;
+  /**
+   * Secondary actions displayed on the right side of the SubNav.
+   * Use SubNavButton and SubNavLink components, which mirror
+   * the DS Button and Link.
+   */
+  secondaryActions?: React.ReactNode;
+}
+
+export const SubNavButton: React.FC<React.PropsWithChildren<any>> = ({
+  id,
+  children,
+  isOutlined,
+  isSelected,
+  screenreaderOnlyText = "",
+}) => {
+  const childrenStyles = useMultiStyleConfig("SubNavChildren", {
+    isOutlined: isOutlined,
+  });
+
+  return (
+    <li>
+      <Button
+        id={id}
+        buttonType="text"
+        className={isSelected ? "selectedItem" : ""}
+        sx={{ ...childrenStyles.outLine }}
+        screenreaderOnlyText={screenreaderOnlyText}
+      >
+        {children}
+      </Button>
+    </li>
+  );
+};
+
+export const SubNavLink: React.FC<React.PropsWithChildren<any>> = ({
+  id,
+  children,
+  isOutlined,
+  isSelected,
+  href,
+  screenreaderOnlyText = "",
+}) => {
+  const childrenStyles = useMultiStyleConfig("SubNavChildren", {
+    isOutlined: isOutlined,
+  });
+
+  return (
+    <li>
+      <Link
+        key={id}
+        id={id}
+        type="action"
+        href={href}
+        isUnderlined={false}
+        screenreaderOnlyText={screenreaderOnlyText}
+        className={isSelected ? "selectedItem" : ""}
+        sx={{ ...childrenStyles.outLine }}
```

Comment:
You should be able to just have sx={childrenStyles.outLine}. It's the same as { ...childrenStyles.outLine }